xml/System.Net.Mail/SmtpClient.xml

```diff
@@ -1640,11 +1640,20 @@ The classes shown in the following table are used to construct email messages th
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="message">A <see cref="T:System.Net.Mail.MailMessage" /> that contains the message to send.</param>
```

Comment:
This can be simplified. The user can already see the data type of the parameter:
```suggestion
        <param name="message">The message to send.</param>
```